### PR TITLE
Questions do not load 121319557

### PR DESCRIPTION
**Pull request by kamellcuba**
@bolerio: I refactored the functions we discuss in the morning. Please review.

### CODE REVIEW

**Comment by kamellcuba** [comment]
@sabbas, @bolerio: is a little big this one (sorry) please review.

File: src/java/org/sharegov/cirm/MetaOntology.java [hunk 1]
```diff
@@ -444,7 +474,7 @@ protected static PropertyDescriptor findPropertyIri(String irifragment) {
 			if (key.equals("label") || key.equals("iri") || key.equals("type"))
 			{
 				if (key.equals("type"))
-					result.add(new AddAxiom(O, factory.getOWLClassAssertionAxiom(owlClass(fullIri(e.getValue().asString())),parent)));
+					result.add(new AddAxiom(O, factory.getOWLClassAssertionAxiom(owlClass(fullIri(PREFIX + e.getValue().asString())),parent)));
```
Comment (sabbas) [inline src/java/org/sharegov/cirm/MetaOntology.java]: PREFIX here is hard coded to legacy could you explain why?

Comment (kamellcuba) [inline src/java/org/sharegov/cirm/MetaOntology.java]: by contract was agreed to perform all operations on "legacy". maybe better include this value on the configuration object and pull it from there so we don't have this hardcoded.

**Comment by kamellcuba** [comment]
@sabbas: can we merge this?

**Pull request merged**